testproject/Assets/Scenes/SampleScene.unity

```diff
@@ -893,13 +883,7 @@ MonoBehaviour:
     AllowRuntimeSceneChanges: 0
     PlayerPrefab: {fileID: 8685790303553767886, guid: 96e0a72e30d0c46c8a5c9a750e8f5807,
       type: 3}
-    NetworkPrefabs:
-    - Override: 0
-      Prefab: {fileID: 442217489085244684, guid: 5eca8a21314fe4278ba2571c289a9773,
-        type: 3}
-      SourcePrefabToOverride: {fileID: 0}
-      SourceHashToOverride: 0
-      OverridingTargetPrefab: {fileID: 0}
+    NetworkPrefabs: []
```

Comment:
did we lose references here?

Reply:
intentional, cleanup from old Unity-Chan! stuff (spawner object and prefab link deleted, this is the prefab link here)